js/private/js_run_devserver.mjs

```diff
@@ -518,6 +518,10 @@ async function main(args, sandbox) {
         // short-circuit for unit tests
         return
     let sandbox
+
+    // Callback to cleanup the sandbox if it exists, once and only once.
+    onProcessEnd(() => sandbox && removeSandbox(sandbox) && (sandbox = null))
```

Comment:
what's the final clause `sandbox = null` doing here? Just freeing the pointer out of fear?

Reply:
To avoid the `fs.rmDir` multiple times. Clear the var on success, only remove if it is set.